lib/new_card.rb

```diff
@@ -18,7 +19,12 @@ def display_confirmed_msg(board_id)
 
     def create_new_card
       data = {
-        'name'=>'tc-i18n-hygiene check failed', 'idList'=>"#{@list_id}", 'due'=>nil
+        'name'=>'tc-i18n-hygiene check failed',
+        'idList'=>"#{@list_id}",
+        'due'=>nil,
+        'desc'=>'https://buildkite.com/conversation/tc-i18n-hygiene',
+        'labels'=>['name'=>'TC','color'=>'green'],
+        'idLabels'=>["#{@id_labels}"]
```

Comment:
@yob and I just discussed this - turns out you don't need labels if you've got `idLabels`, as the id contains specific attributes